pro/src/core/Offerers/adapters/getOffererNamesAdapter.ts

```diff
@@ -24,4 +23,7 @@ export const getOffererNamesAdapter: TGetOffererNamesAdapter = async () => {
   }
 }
 
+export const useGetOffererNames = () =>
+  useAdapter<TOffererName[], TOffererName[]>(getOffererNamesAdapter, [])
```

Comment:
Est ce qu'il y a une réelle utilité à passer par un adapter ?
Ici tu peux donner directement ta fonction `apiV1.getOfferersListOfferersNames` (et changer le hook pour qu'il fasse le try/await/catch de l'adapter) plutôt que de passer systématiquement par un intermédiaire

```suggestion
export const useGetOffererNames = () =>
  useFetcher<TOffererName[]>(apiV1.getOfferersListOfferersNames, [])
```

Reply:
Oui ce nouveau hook fait doublons. Jusque là les adapters se charge de faire la conversion du format de données de l'api vers celui attendu par la le screen. 
Et les effect sont généré dans des useEffect de la route, c'est le role de la route.

Avec cette demande de gérer les effect dans un hook, la fonction de l'adapter tel qu'elle est peu y etre mergé.
Ca peu aussi ouvrir des questions sur l'utilité des routes dans ce context.